code/socialDistribution/urls.py

```diff
@@ -1,6 +1,15 @@
 from django.urls import path
+from django.contrib import admin
 from . import views
 
 urlpatterns = [
-  path('', views.index, name='index')
+  path('', views.home, name='home'),
+  path('home/', views.home, name='home'),
```

Comment:
Why do we have two paths called "home"? I think it would be best to leave the ' ' path as name="index". 

Similarly, in views.py, I think we should keep both index() and home() functions. Index() can return some generic text for now. Later, we can have index() redirect the client to "/home" if the proper conditions are met (instead of just sending the same HTML for two different paths)

Reply:
Sounds good